alembic/versions/522d776a42ed_event_models.py

```diff
@@ -0,0 +1,100 @@
+"""event_models
+
+Revision ID: 522d776a42ed
+Revises: 447728ca6d2e
+Create Date: 2015-04-15 01:31:51.056264
+
+"""
+
+revision = '522d776a42ed'
+down_revision = '447728ca6d2e'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table('ticket_type',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=False),
+        sa.Column('name', sa.Unicode(length=80), nullable=False),
+        sa.Column('proposal_space_id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(['proposal_space_id'], ['proposal_space.id'], ),
+        sa.PrimaryKeyConstraint('id')
+        )
+    op.create_table('event',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=False),
+        sa.Column('name', sa.Unicode(length=80), nullable=False),
+        sa.Column('proposal_space_id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(['proposal_space_id'], ['proposal_space.id'], ),
+        sa.PrimaryKeyConstraint('id')
+        )
+    op.create_table('participant',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=False),
+        sa.Column('fullname', sa.Unicode(length=80), nullable=False),
+        sa.Column('email', sa.Unicode(length=80), nullable=False),
```

Comment:
Email column should be length 254 to qualify against the official spec (which defines it as up to 256 characters including a `<>` frame).